apps/vx-scan/backend/src/app.ts

```diff
@@ -222,9 +221,8 @@ function buildApi(
       store.setBallotCountWhenBallotBagLastReplaced(store.getBallotsCounted());
     },
 
-    async backupToUsbDrive(): Promise<Result<void, Scan.BackupError>> {
-      const result = await backupToUsbDrive(store, usb);
-      return result.isErr() ? result : ok();
+    async backupToUsbDrive(): Promise<Result<void, ExportDataError>> {
+      return await backupToUsbDrive(store, usb);
```

Comment:
I moved this line into `backupToUsbDrive` so it would match `exportCastVoteRecordsToUsbDrive`